LoopKitUI/Views/SelectableLabel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SelectableLabel.swift
+//  LoopKitUI
+//
+//  Created by Nathaniel Hamming on 2020-02-20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct SelectableLabel: View {
+    var label: String
+    @Binding var selectedLabel: String?
+    
+    public init(label: String, selectedLabel: Binding<String?>) {
+        self.label = label
+        _selectedLabel = selectedLabel
+    }
+    
+    public var body: some View {
+        Button(action: { self.selectedLabel = self.label }) {
+            HStack {
+                Text(label)
+                    .foregroundColor(.primary)
+                Spacer()
+                Image(systemName: selectedLabel == label ? "checkmark.circle.fill" : "")
```

Comment:
[nit] Rather than pass an empty string so that `Image(systemName:)` fails, it would be clearer to have

```swift
if selectedLabel == label {
    Image(systemName: "checkmark.circle.fill")
}
```